sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedResponseBase.java

```diff
@@ -62,8 +63,10 @@ public PagedResponseBase(HttpRequest request, int statusCode, HttpHeaders header
      * {@inheritDoc}
      */
     @Override
-    public List<T> getItems() {
-        return items;
+    public IterableStream<T> getElements() {
+        return items == null
+            ? null
```

Comment:
Prefer to return empty IterableStream as [described here](https://github.com/Azure/azure-sdk-for-java/pull/6465/files?file-filters%5B%5D=.java&file-filters%5B%5D=.xml#r363119004)